kpet/data.py

```diff
@@ -381,13 +357,6 @@ def __init__(self, data):
         )
         if self.pattern is None:
             self.pattern = Pattern({})
-        # TODO Remove once database transitions to names
-        if self.name is None:
-            if self.description is None:
-                raise Invalid(
-                    'The suite has neither name nor description specified.')
-            self.name = self.description
-        self.validate_case_ids()
```

Comment:
So, the suites don't have an id or description, and the name is optional? Is that correct?

Reply:
Yep. I have a commit verifying that the compound names (suite + case names) are unique, and that should serve as ID for the data warehouse.